lang/en/rocketchat.php

```diff
@@ -104,12 +104,12 @@
 $string['rcgrouperror'] = 'Remote Rocket.Chat group can\'t be retrieved. Please contact your system administrator. Error code {$a}.';
 $string['usernamehook'] = 'Activate username hook.';
 $string['usernamehook_desc'] = 'By activating this option it will be possible to chang moodle username to fit Rocket.Chat username.</br>Create a hooklib.php file in the module rocketchat installation directory.</br>code a moodle_username_to_rocketchat function that will return the transformed moodle username to fit Rocket.Chat one.</br>hooklib-example.php is given as example.';
-$string['background_enrolment_task'] = 'Pass user enrolment/unenrolments in remote Rocket.Chat as background tasks.';
-$string['background_enrolment_task_desc'] = 'These to solve performance issue while enrolling large amounts of users.</br>This will prevent the enroller to wait on course enrolment page while enrolling/unenrolling large amount of users.</br>Choice of enrol cohort and flatfile, if enabled, are strongly advised';
+$string['background_enrolment_task'] = 'Pass user enrolment/unenrolments to remote Rocket.Chat as background tasks.';
+$string['background_enrolment_task_desc'] = 'This is to solve performance issue while enrolling large amounts of users.</br>This will prevent the enroller to wait on course enrolment page while enrolling/unenrolling large amount of users.</br>Choice of enrol cohort and flatfile, if enabled, are strongly advised';
 $string['background_add_instance'] = 'Pass Rocket.Chat enrolments to background task while creating a new module instance';
-$string['background_add_instance_desc'] = 'This will prevent to wait on the module creation page';
+$string['background_add_instance_desc'] = 'This will prevent waiting of the module creation page';
```

Comment:
I think the correct preposition is "for".
This will prevent waiting for the module creation page

Reply:
For is used to indicate the reason or 'because of', while here, 'of' is used as a function word to indicate the component material, parts, or elements or the contents

Reply:
OK. Thanks for the explanation.